### ISSUE TITLE
Extraneous Log Text in Report

### ISSUE THREAD

**Issue by ericthomasswenson** [Equipment-and-Tool-Institute/j1939-84#1105]
Extraneous log text observed in report.

End Test 1.8 - DM20: Monitor Performance Ratio

Start Test 1.9 - Component ID: Make
14:31:52.6212 Destination Specific **org.etools.j1939tools.j1939.packets.ComponentIdentification**Packet Request to Engine #1 (0)
14:31:52.6265 18EA00F9 [3] EB FE 00 (TX)
14:31:52.7999 1CFEEB00 [49] 49 4E 54 20 20 2A 37 35 37 31 32 34 32 30 32 31 2A 58 58 58 58 58 58 58 58 58 58 58 58 58 58 2A 30 30 30 30 30 30 30 30 30 30 30 30 30 30 30 30 2A
Component Identification from Engine #1 (0): {
  Make: INT
  Model: 7571242021
  Serial: XXXXXXXXXXXXXX
  Unit: 0000000000000000
}

**Comment by ericthomasswenson** [comment]
Independent Observation

Start Test 1.9 - Component ID: Make
11:47:14.6090 Destination Specific **_org.etools.j1939tools.j1939.packets.ComponentIdentificationPacket_** Request to Engine **_ https://github dot com/Equipment-and-Tool-Institute/j1939-84/pull/1_** (0)
11:47:14.6115 18EA00F9 [3] EB FE 00 (TX)
11:47:14.7310 1CFEEB00 [39] 44 54 44 53 43 2A 34 37 32 4E 31 39 2A 34 37 32 39 31 32 53 34 37 32 39 31 32 53 39 39 39 39 39 39 39 39 39 39 39 39 39 39 2A 20 20 20 20 20 20 20 20 20 20 20 20 20 20 20 20 20 2A
Component Identification from Engine **_https://github.com/Equipment-and-Tool-Institute/j1939-84/pull/1_** (0): {
Make: DTDSC
Model: 472N19
Serial: 472912S9999999
Unit:
}

**Comment by jacobbosch-cummins** [comment]
Issue observed with Cummins logs as well.

**Comment by battjt** [comment]
Test scafolding was unintentionally used by the production logic.
```
public class CommunicationsModule extends FunctionalModule {
...
   public <T extends GenericPacket> BusResult<T> request(Class<T> clazz,
                                                          int address,
                                                          CommunicationsListener listener) {
        int pg = getPg(clazz, listener);
        Packet requestPacket = getJ1939().createRequestPacket(pg, address);
        return getJ1939().requestDS(clazz.getName(), pg, requestPacket, listener);
    }
```

**Comment by jacobbosch-cummins** [comment]
Garbled test for CI message is still present on 3.1.3.  

Start Test 1.9 - Component ID: Make
15:23:18.5149 Destination Specific org.etools.j1939tools.j1939.packets.ComponentIdentificationPacket Request to Engine #1 (0)
15:23:18.5169 18EA00F9 [3] EB FE 00 (TX)

**Comment by ericthomasswenson** [comment]
Closed in favor of #1164 with two remaining examples.
